### ISSUE TITLE
Ensure that all Android apps support 64bit

### ISSUE THREAD

**Issue by garethbowen** [medic/medic#5642]
Our apps may be delisted from the Play Store if we don't support 64 bit architectures before the 1st of August 2019. Follow [these instructions](https://developer.android.com/distribute/best-practices/develop/64-bit) to check and fix any issues for...

- [x] medic-android
- [x] medic-gateway
- [x] medic-collect
- [x] rdt-capture

**Comment by garethbowen** [comment]
@henokgetachew patched one app to get 64 bit support: https://github.com/medic/medic-projects/issues/6329#issuecomment-497716251

**Comment by henokgetachew** [comment]
More updates about medic-android.  Crosswalk is not being maintained at the moment and the builds they have available are one for 32bit and another for 64bit but no build for `arm64-v8a` meaning right now, we can't support both at the same time.

Here are the options:
1.  Dropping 32bit device support for medic-android and serving only 64bit - this is not operationally practical.
2. Trying to create that lib on our own.  Either: 
-      Building the code on our own.
-      Manually merging the two libs and including the resulting lib in our repo - This sounds bad but would take the least amount of time and it's a one time thing.
3. Having two apps on the play store - one for 32bit and another for 64bit devices.  This is not ideal because it would essentially double the number of apps on the playstore that we would have to maintain and complicate our build process.
I'm experimenting with the second option right now.

**Comment by henokgetachew** [comment]
I have got it working by merging the two crosswalk libs. I will clean it up and push the code.

**Comment by henokgetachew** [comment]
Medic-android now supports both 32bit and 64bit architectures.  It has been QA'd on various devices by @kitsao

**Comment by henokgetachew** [comment]
The new crosswalk lib was crafted by merging the 32bit and 64bit versions of crosswalk available [here](https://download.01.org/crosswalk/releases/crosswalk/android/maven2/org/xwalk/xwalk_core_library/23.53.589.4/).  It's right now found in medic-android repo under `libs/xwalk_core_library-23.53.589.4-arm64-v8a.aar`

**Comment by henokgetachew** [comment]
All of the apps above support both architectures.  We can consider this as already AT'd because @kitsao has been testing on the field but I will leave that decision for @MaxDiz.  I'm moving it to AT.

**Comment by MaxDiz** [comment]
@wtekeu is release manager this cycle - your call on AT procedure
cc: @garethbowen

**Comment by derickl** [comment]
From what was done here https://github.com/medic/medic/issues/5642#issuecomment-499838852,
we've ended up almost doubling the APK size - from 25ish MB to almost 50MB

**Comment by derickl** [comment]
Based on this doc https://developer.android.com/studio/build/configure-apk-splits, it should be possible to build multiple versions of the same app and submit those.

Quoting from the link above

```
Although you should build a single APK to support all your target devices whenever possible, that might result in a very large APK due to files needed to support multiple screen densities or Application Binary Interfaces (ABIs). One way to reduce the size of your APK is to create multiple APKs that contain files for specific screen densities or ABIs. 
```

**Comment by garethbowen** [comment]
Also:

> By publishing your application with multiple APKs, you can: Support different CPU architectures with each APK (such as for ARM or x86...)

https://developer.android.com/google/play/publishing/multiple-apks

**Comment by garethbowen** [comment]
I've raised a new issue to fix the apk size issue so this one can just focus on supporting a wide range of phones.

**Comment by MaxDiz** [comment]
@henokgetachew following #6222 Muso requires an upgrade for near-term trial rollout. Do you want a new ticket created?
cc: @Fatoufall

**Comment by henokgetachew** [comment]
Closing this as taken care of.  @MaxDiz re: Muso, as soon as the new feature is ready, we would just need to tag a release and it would be automatically updated. So, whoever is working on the other ticket just needs to tag a new release once they're done.

**Comment by MaxDiz** [comment]
@wtekeu @kennsippell see ^^ from @henokgetachew re 64-bit support for #5695 and other mRDT work to be released in v3.6
cc: @Fatoufall
